### PR TITLE
Whole TextView padding between lines

### LINKED ISSUE #306: Whole TextView padding between lines
## Kind (Required)

- Improvement, discussion

 ## Overview (Required)

- I think we should increase the spacing between lines in whole textview.

current | figma
---- | ----
 | 

 ## Links	

- https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=21%3A395
- https://developer.android.com/reference/android/widget/TextView.html#attr_android:lineSpacingExtra

### PR DESCRIPTION
**Pull request by koogawa**
## Issue
- close #306 

## Overview (Required)
- I think we should increase the spacing between lines in whole textview.

## Links

- https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=21%3A395
- https://developer.android.com/reference/android/widget/TextView.html#attr_android:lineSpacingExtra

### Timeline

Before | After | Figma
:--: | :--: | :--:
 |  | 

## Session

Before | After | Figma
:--: | :--: | :--:
 |  | 

## Speaker

Before | After | Figma
:--: | :--: | :--:
 |  |

### CODE REVIEW

**Comment by mhidaka** [comment]
👀

File: feature/session/src/main/res/layout/item_session.xml [hunk 1]
```diff
@@ -80,6 +80,7 @@
             android:singleLine="false"
             android:textAppearance="?attr/textAppearanceHeadline6"
             android:textColor="@color/session_title"
+            android:lineSpacingExtra="8sp"
```
Comment (mhidaka) [inline feature/session/src/main/res/layout/item_session.xml]: I guess session title need more spacing on figma. Maybe it should be `16sp` or `12sp` ?

Comment (mhidaka) [reply feature/session/src/main/res/layout/item_session.xml]: @koogawa Could you tweak it as a new Pull Request?

Comment (koogawa) [reply feature/session/src/main/res/layout/item_session.xml]: That's a good point!

**Pull request merged**